week1/project/hackyourweather/src/cityList.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import City from './city';
+import data from './city-weather.json'
+function CityList(){
+    return (
+        <ul className="city-list">
+{data.map(city=>{
+    return <City city={city}/>
+})}
```

Comment:
one suggestion and one note,
the suggestion, is that you don't need three lines for this small `.map` method, and you can write it in one of two ways:
```suggestion
{data.map(city=> <City city={city}/>)}
```
or if you have long input or you prefer multiline:
```suggestion
{data.map(city=>(
  <City city={city}/>
))}
```

the note is about the key, when you open the console, you will notice a warning about something related to "key" stuff, the key in react is usually used to identify the top level component/element in an array, in your case it's `<City />` component, it has a use, and you will probably be introduced to it in your next lesson.
```suggestion
{data.map(city=>{
    return <City key={city.id} city={city}/>
})}
```

---
> if you don't have an id in your object you can use [uuid](https://www.npmjs.com/package/uuid) instead